ci/scripts/driver_weekly.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+set -eux
+
+##############################################################################################
```

Comment:
I just want to make sure we note that this script only needs to run on a single machine and not on every machine the CI is being run on.  This is so because all this script does is create a branch push it to the users' fork and apply CI triggering labels.

Reply:
Done.  It's weird that ChatGPT knew enough about the inner workings of the script to finish my sentences  when I was adding this into the comment section.